plugin/src/cli_utils/localpvhostpath/mod.rs

```diff
@@ -0,0 +1,77 @@
+use clap::Parser;
+use plugin::ExecuteOperation;
+use crate::cli_utils::CliArgs;
+
+/// LocalPV Hostpath operations.
+#[derive(Parser, Debug)]
+pub(crate) enum Operations {
+    // Getter for localpv-hostpath resources.
+    #[clap(subcommand)]
+    Get(HosthpathGetter)
+}
+
+#[async_trait::async_trait(?Send)]
+impl ExecuteOperation for Operations {
+    type Args = CliArgs;
+    type Error = Error;
+
+    async fn execute(&self, cli_args: &CliArgs) -> Result<(), Error> {
+        match self {
+            Operations::Get(hostpathgetter) => {
+                hostpathgetter.execute(cli_args).await?;
+            }
+        }
+        Ok(())
+    }
+}
+
+#[derive(clap::Subcommand, Debug)]
+pub(crate) enum HosthpathGetter {
+    /// Gets a specific localpv-hostpath volume.
+    Volume(GetVolumeArg),
+    /// Lists all localpv-hostpath volumes.
+    Volumes(GetVolumesArg),
+}
+
+/// Arguments used when getting a hostpath volume.
+#[derive(Debug, Clone, clap::Args)]
+pub struct GetVolumeArg {
+    /// Volume id of the volume.
+    volume: String
+}
+
+/// Arguments used when listing hostpath volumes.
+#[derive(Debug, Clone, clap::Args)]
+pub struct GetVolumesArg {
```

Comment:
GetVolumesArgs

Reply:
This struct has one field now. I dont see it getting more. Lets keep Arg for now?

Reply:
Sure, upto you. But incase we have more in future it will again need changes. But that's subjective.